doc/src/vpr/file_formats.rst

```diff
@@ -738,6 +738,73 @@ An example listing for a global net is given below.
     Block pksi_185_ (#432) at (5,48), pinclass 2
     Block n_n2879 (#433) at (49,23), pinclass 2
 
+.. _vpr_flat_placement_file:
+
+Flat Placement File Format (.flat_place)
+----------------------------------------
+The flat placement file contains the information needed for VPR's legalizer to reconstruct a clustering and placement solution,
+inferring mode and intra-cluster routing information and correcting legality violations as needed. 
```

Comment:
I'd add: "This file is an alternative input to the packing (.net) and clustered placement (.place) files; the more succinct and higher-level input format allows an external tool to more easily provide a packing and placement solution to VPR."